jenkins-x-lint.yml

```diff
@@ -22,4 +22,13 @@ pipelineConfig:
               - setup_linter
               - run_linter_check
               command: make
+            - name: pip-install-flake8
+              command: python3
+              args:
+              - -m pip install flake8
+            - name: python-lint-with-flake8
+              command: flake8
+              args:
+              - . --count --select=E9,F63,F7,F82 --show-source --statistics
```

Comment:
It could also be good to move the `flake8` config into the `setup.cfg` file (or a `.flake8` one), that way everyone also shares the same config locally.